akka-docs/src/main/paradox/cluster-usage.md

```diff
@@ -1,154 +1,51 @@
-# Cluster Usage
-
-For introduction to the Akka Cluster concepts please see @ref:[Cluster Specification](common/cluster.md).
+# Classic Cluster Usage
+ 
+This document describes how to use Akka Cluster and the Cluster APIs using code samples.
+For specific documentation topics see:  
 
-The core of Akka Cluster is the cluster membership, to keep track of what nodes are part of the cluster and
-their health. There are several @ref:[Higher level Cluster tools](cluster-usage.md#higher-level-cluster-tools) that are built
-on top of the cluster membership.
+* @ref:[Cluster Specification](typed/cluster-specification.md)
+* @ref:[User Guide](typed/cluster.md)
+* @ref:[Cluster Membership Service](typed/cluster-membership.md)
+* @ref:[When and where to use Akka Cluster](typed/choosing-cluster.md)
+* @ref:[Rolling Updates](additional/rolling-updates.md)
+* @ref:[Operating, Managing, Observability](additional/operations.md)
 
-You need to enable @ref:[serialization](serialization.md) for your actor messages.
-@ref:[Serialization with Jackson](serialization-jackson.md) is a good choice in many cases and our
-recommendation if you don't have other preference.
+Enable @ref:[serialization](serialization.md) to send events between ActorSystems and systems
+external to the Cluster. @ref:[Serialization with Jackson](serialization-jackson.md) is a good choice in many cases, and our
+recommendation if you don't have other preferences or constraints.
 
 ## Dependency
 
-To use Akka Cluster, you must add the following dependency in your project:
+To use Akka Cluster add the following dependency in your project:
 
 @@dependency[sbt,Maven,Gradle] {
   group="com.typesafe.akka"
   artifact="akka-cluster_$scala.binary_version$"
   version="$akka.version$"
 }
 
-## Sample project
+## Cluster samples
```

Comment:
Inconsistent casing of headers, some are leading caps and some are all caps, perhaps that is a leftover from old docs but would ben nice if consistent at least within PR changes

Reply:
Ya, we have that literally all over the place. Each page has its own header case convention. So I've been picking the primary one per page and going with that. Do we want to start one now?

Reply:
Created https://github.com/akka/akka/issues/27741 to track instead.